src/main/java/com/example/bookstoreapp/model/CartItem.java

```diff
@@ -0,0 +1,29 @@
+package com.example.bookstoreapp.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
```

Comment:
Do u use Soft delete for cartItem class? i thought that is the part of our task

Reply:
As I remember Shopping cart may have soft delete. Cart item with soft delete will fill database too quickly